src/JSURI.js

```diff
@@ -10,15 +10,15 @@ function toRFC3896(input) {
   });
 }
 
-exports._encodeURIComponent = function encode(fail, succeed, input) {
+exports._encodeURIComponent = function _encodeURIComponent(fail, succeed, input) {
   try {
     return succeed(toRFC3896(encodeURIComponent(input)));
   } catch (err) {
     return fail(err);
   }
 };
 
-exports._encodeFormURLComponent = function encode(fail, succeed, input) {
+exports._encodeFormURLComponent = function _encodeFormURLComponent(fail, succeed, input) {
   try {
     return succeed(toRFC3896(encodeURIComponent(input)).replace(/%20/g, "+"));
```

Comment:
While you're here, do you mind changing all instances of `RFC3896` to be `RFC3986`? It's a typo; [the actual RFC is 3986](https://web.archive.org/web/20210118083445mp_/https://tools.ietf.org/html/rfc3986).